eng/dockerfile-templates/aspnet/Dockerfile.windows

```
@@ -1,20 +1,28 @@
 {{
     set dotnetVersion to join(slice(split(PRODUCT_VERSION, "."), 0, 2), ".") ^
-    set baseUrl to VARIABLES[cat("dotnet|", dotnetVersion, "|base-url|", VARIABLES["branch"])] ^
+    set isServerCore to find(OS_VERSION, "windowsservercore") >= 0 ^
+
+    set dotnetBaseUrl to VARIABLES[cat("dotnet|", dotnetVersion, "|base-url|", VARIABLES["branch"])] ^
     set isInternal to find(baseUrl, "artifacts.visualstudio.com") >= 0 ^
     set isSingleStage to (find(OS_VERSION, "windowsservercore") >= 0 && !isInternal) ^
-    set tagVersion to when(dotnetVersion = "6.0" || dotnetVersion = "8.0",
-        VARIABLES[cat("dotnet|", dotnetVersion, "|product-version")]
-        VARIABLES[cat("dotnet|", dotnetVersion, "|fixed-tag")]) ^
-    set runtimeBaseTag to cat("$REPO:", tagVersion, "-", OS_VERSION)
+
+    set repoBaseUrl to "mcr.microsoft.com/windows" ^
+    set installerStageRepo to cat(repoBaseUrl, "/servercore") ^
+    set installerStageTag to cat(OS_VERSION_NUMBER, "-amd64") ^
+
+    set installerStageFromImage to cat(installerStageRepo, ":", installerStageTag) ^
```

Most of these variables don't seem to provide any value. This can be simplified:

```suggestion
    set installerStageFromImage to cat("mcr.microsoft.com/windows/servercore:", OS_VERSION_NUMBER, "-amd64") ^
```

This comment applies to the other image type templates as well.

---

Yes these were used when Server 2025 was insider-only and had a different repo/tag. Fixed in https://github.com/dotnet/dotnet-docker/pull/5944/commits/9ba744d6a66be79e76c3c99e538a1049a1c22892.